tools/generate_hotkey.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
+
+
+general = []
+msg_actions = []
+msg_compose = []
+navigation = []
+searching = []
+stream_list = []
+
+CATEGORY = {
+    'general': general,
+    'msg_actions': msg_actions,
+    'msg_compose': msg_compose,
+    'navigation': navigation,
+    'searching': searching,
+    'stream_list': stream_list,
+}
+
+for key, item in KEY_BINDINGS.items():
+    CATEGORY[item['key_category']].append((item['help_text'], item['keys']))
+
+OUTPUT_FILE = "docs/hotkey.md"
+mdFile = open(OUTPUT_FILE, "w")
+mdFile.write("<!--- Generated automatically by tools/generate_hotkey.py -->\n")
+mdFile.write("<!--- Do not modify -->\n\n")
+mdFile.write("# HOT KEYS\n")
+
+actions = ['general', 'navigation', 'searching', 'msg_actions', 'stream_list', 'msg_compose']
+for action in actions:
+    mdFile.write("## {}\n".format(HELP_CATEGORIES[action]))
+    mdFile.write("|Command|Key Combination|\n")
+    mdFile.write("| :---: | :---: |\n")
+    for help_text, key_combinations_list in CATEGORY[action]:
+        various_key_combinations = []
+        for key_combination in key_combinations_list:
+            new_key_list = key_combination.split()
+            for i in range(len(new_key_list)):
+                new_key_list[i] = "<kbd>"+new_key_list[i]+"</kbd>"
+            new_key_list = " + ".join(new_key_list)
+            various_key_combinations.append(new_key_list)
+        various_key_combinations = " / ".join(reversed(various_key_combinations))
+        mdFile.write("|{}|{}|\n".format(help_text, various_key_combinations))
+    mdFile.write("\n\n")
+
+mdFile.write("**Note:** You can use `arrows`, `home`, `end`,"
+                        "`Page up` and `Page down` keys to move around"
+                        "in Zulip-Terminal.")
```

Comment:
This may be unnecessary given the section with navigation?

Reply:
I left it there as it was there in the README file.

Reply:
I see you removed it - that's great, though let's ensure that's noted in the commit text.